internals-js/src/error.ts

```diff
@@ -513,6 +513,12 @@ const OVERRIDE_ON_INTERFACE = makeCodeDefinition(
   { addedIn: '2.3.0' },
 );
 
+const OVERRIDE_LABEL_INVALID = makeCodeDefinition(
+  'OVERRIDE_LABEL_INVALID',
+  'The @override directive `label` argument must match the pattern /^[a-zA-Z][a-zA-Z0-9_\-:./]*$/ or /^percent\(([0-9]|[1-9][0-9]|100)\)$/',
```

Comment:
Need to update the regex for the `percent` case here.

Reply:
c0a3638452b8fbcf7407c847e60a8ec8d120451d